modules/lymphgen/1.0/config/default.yaml

```diff
@@ -3,18 +3,53 @@ lcr-modules:
+            sample_maf: "__UPDATE__" 
+            sample_seg: 
+            # Each key below can be the file path, with wildcards, for the output of a CNV file produced by a specific cnv caller (eg. battenberg)
+            # The order of the keys determine what which file paths to prioritize first
+            #       eg. battenberg: "results/battenberg-1.2/99-outputs/seg/{seq_type}--projection/{tumour_id}--{normal_id}--{pair_status}.{tool}.{projection}.seg"
+            #           sequenza: "results/sequenza-1.4/99-outputs/seg/{seq_type}--projection/{tumour_id}--{normal_id}--{pair_status}.{tool}.{projection}.seg"
+            #    It will first look for that sample's battenberg cnv file and then sequenza 
+            # Only uncomment the keys that have file paths that exist (always keep "empty" uncommented)
+            # Add any additional keys before "empty"
+            # If you don't have an CNV data, just include "empty" (an empty seg file), and the module will run without cnv data 
+                # battenberg: "__UPDATE__"
+                # sequenza: "__UPDATE__"
+                # controlfreec: "__UPDATE__"
+                # cnvkit: "__UPDATE__"
+                empty: "{MODSDIR}/etc/empty.seg" 
+            sample_sv_info: 
+            # All the SV files will be merged with a final SV file created for each sample so the order of keys don't matter 
+            # If there is no fish sv file put "{MODSDIR}/etc/empty_sv_fish.tsv" in key fish 
+            # If there is no svar_master file put "{MODSDIR}/etc/empty_svar_master.tsv" under both empty_svar and svar_master keys
+            # Always keep all three keys with real file paths 
+                fish: "__UPDATE__"
+                empty_svar: "{MODSDIR}/etc/empty_svar_master.tsv" 
+                svar_master: "__UPDATE__" 
             # Path to the directory where LGenIC should be installed
             lgenic_exec: "__UPDATE__"
+            gamblr_branch: "" # Leave blank unless you need to install from a specific branch from the GAMBLR repo (expert users only)
+            gamblr_config_url: "https://raw.githubusercontent.com/morinlab/GAMBLR/master/config.yml"
+        
+        pairing_config:
+            genome:
+                run_paired_tumours: True
+                run_unpaired_tumours_with: "unmatched_normal"
+                run_paired_tumours_as_unpaired: False
+            capture:
+                run_paired_tumours: True
+                run_unpaired_tumours_with: "unmatched_normal"
+                run_paired_tumours_as_unpaired: False
+        
+        unmatched_normal_ids:
+            capture--grch37: "__UPDATE__"
+            # Add additional {seq_type}--{genome_build}
 
         options:
-            outprefix: "my_cohort"
             reformat_seg:  #  Default columns for Battenberg output
                 Tumor_Sample_Barcode: "ID"
-                chromosome: "chrom"
+                chromosome: "chr"
```

Comment:
Is this how the CNV modules encode the header for the seg files? It's `ID chr start end` not `ID chrom start end`?